src/client.ts

```diff
@@ -1,24 +1,52 @@
 import { Client, Intents } from "discord.js";
+import { config } from "dotenv";
 import { Db } from "mongodb";
 
 import cmd_register from "./commands/register";
 import {
   cmd_tenmans,
   handleButton as tenmansHandleButton,
 } from "./commands/tenmans";
+import { cmdConfig } from "./commands/config";
+import botConfig from "./config/botConfig";
+
+async function initBotConfig (client: Client, db: Db) {
+  const botConfigCollection = db.collection("config");
+  let botConfigDoc = await botConfigCollection.findOne({
+    "configName": "bot"
+  });
+    
+  if (!botConfigDoc) {
+    // Persist default configuration data
+    const defaultChannelId = process.env.DEFAULT_QUEUEMSG_CHANNELID || "885704092142428200";
```

Comment:
After all of this, will there be any need for this hardcoded string? That was a temporary measure until someone made it configurable

Reply:
If someone forgets to set the env var, I figure having a fallback is useful. Otherwise, I'm worried that the bot would hard break without error handling. We could add the error handling around the channel, but this strikes a nice convenience in my opinion.

Reply:
That being said, it's not very convenient if you're not using the bot in our server. So perhaps it's better to add error handling.